frontend/src/app/application-forms/_services/application-fields.service.ts

```diff
@@ -131,32 +131,33 @@ export class ApplicationFieldsService {
     }
   }
 
-  scrollToFirstError() {
-    const invalidElements = document.querySelectorAll(
-      'input.ng-invalid, select.ng-invalid, textarea.invalid, .usa-file-input.ng-invalid, .ng-untouched.required'
-    );
-    if (invalidElements.length === 0) {
-      return;
-    }
-    invalidElements[0].scrollIntoView();
-    return this.getInvalidElement(invalidElements);
-  }
-
-  getInvalidElement(invalidElements) {
-    let invalid = document.getElementById(invalidElements[0].getAttribute('id'));
-    if (!invalid) {
-      const invalidClass = document.getElementsByClassName(invalidElements[0].getAttribute('class'));
+  private getInvalidElement(firstInvalidElement) {
```

Comment:
is this for scrolling back to the validation error? mind adding a typedoc annotation for this method? 

also will this affect how tempOutfitter/ nonCommericial work?

Reply:
This will not affect tempOutfitter/nonComm. I added typedoc that explains the function.